src/main/java/frc5124/robot2020/subsystems/Turret.java

```diff
@@ -99,6 +95,22 @@ public void disableTurretPID () {
     turretPID.setD(0);
   }
 
+  public void setHome(boolean isHome) {
+    this.isHome = isHome;
+  }
+
+  public boolean setHome () {
+    return isHome;
+  }
+
+  public void initialHome(boolean initialHome) {
```

Comment:
getters and setters are more clearly identified when we use get and set in their names